prow/jenkins/jenkins.go

```diff
@@ -261,14 +289,18 @@ func (c *Client) BuildFromSpec(spec *kube.ProwJobSpec, buildId string) error {
 // provided jobs (both scheduled and enqueued).
 func (c *Client) ListBuilds(jobs []string) (map[string]JenkinsBuild, error) {
 	// Get queued builds.
+	started := time.Now()
 	jenkinsBuilds, err := c.GetEnqueuedBuilds(jobs)
+	c.measure("GetEnqueuedBuilds", started, err)
```

Comment:
Why is this not in the `GetEnqueuedBuilds` call itself?

Reply:
I need the error from the call and I wanted to avoid `defer`ing the error check inside the call.

Reply:
I am still hoping we find a solution for logging and metrics that wraps cleanly around the lowest level calls to HTTP transport